src/main/resources/db/migration/V7__create_applicant_user_role.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `UserRoles`(`userRoleName`) VALUES ('APPLICANT');
```

Comment:
Is this value correct? In your map, the value is "user", don't we want to keep them consistent?

Reply:
fixed and changed to be consistent